modules/mavlink

```diff
@@ -1 +1 @@
-Subproject commit 4e086b51a20c0d09eeb095d012a0fcaccf079d6e
```

Comment:
Hi I've done a rebase. No changes were made to a module apart from updating them. Could you check again please.